client/src/components/DogParkShow.js

```diff
@@ -34,9 +41,48 @@ const DogParkShow = (props) => {
         return <p className="tag-cloud-individual-tag">{tag}</p>
     })
 
+    const deleteReview = async (reviewId) => {
+        try {
+            const response = await fetch(`/api/v1/parks/${parkId}/reviews/${reviewId}`, { method: "DELETE"})
```

Comment:
when deleting a review we only need to know the id of the review to delete - parks is not needed in the delete request
```js
`/api/v1/reviews/${reviewId}`
```